### PR TITLE
CORE-6748 config RPC endpoint doc update

### PR DESCRIPTION
**Pull request by LWogan**

### CODE REVIEW

**Review by vkolomeyko — CHANGES_REQUESTED** [review]
These changes need to be targetting DP2 branch.

File: processors/rpc-processor/src/integrationTest/resources/swaggerBaseline.json [hunk 1]
```diff
@@ -3420,7 +3420,7 @@
             "example" : 0
           }
         },
-        "description" : "Details of the updated configuration"
+        "description" : "Details of the updated configuration. Includes: \n- config: Updated configuration in JSON or HOCON format.\n- schemaVersion: Schema version of the configuration.\n- section: Section of the configuration to be updated.\n- version: Version number used for optimistic locking.\n"
```
Comment (paulmoloneyr3) [inline processors/rpc-processor/src/integrationTest/resources/swaggerBaseline.json:3423]:
```suggestion
        "description" : "Details of the updated configuration. Includes: \n- config: The updated configuration in JSON or HOCON format.\n- schemaVersion: The schema version of the configuration.\n- section: The section of the configuration to be updated.\n- version: The version number used for optimistic locking.\n"
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]
Minor changes

File: processors/rpc-processor/src/integrationTest/resources/swaggerBaseline.json (hunk 1 above)
Comment (LWogan) [reply processors/rpc-processor/src/integrationTest/resources/swaggerBaseline.json:3423]: ive applied these changes to a new pr pointing at the dev preview 2 branch

**Comment by LWogan** [comment]
https://github.com/corda/corda-runtime-os/pull/2166
new PR pointing at dev preview 2

**Pull request closed without merging**